sbol_utilities/sbol3_genbank_conversion.py

```diff
@@ -206,15 +204,129 @@ def convert_genbank_to_sbol3(
+        map_created = self.create_GB_SO_role_mappings(
+            so2gb_csv=SO2GB_MAPPINGS_CSV, convert_gb2so=False
+        )
+        if not map_created:
+            # TODO: Need better SBOL3-GenBank specific error classes in future
+            raise ValueError(
+                f"Required CSV data files are not present in your package.\n    Please reinstall the sbol_utilities package.\n \
+                Stopping current conversion process.\n    Reverting to legacy converter if new Conversion process is not forced."
+            )
+        # consider sbol3 objects which are components
+        logging.info(f"Parsing SBOL3 Document components using SBOL3 Document: \n{doc}")
+        for obj in doc.objects:
+            if isinstance(obj, sbol3.Component):
+                logging.info(f"Parsing component - `{obj.display_id}` in sbol3 document.")
+                # NOTE: A single component/record cannot have multiple sequences
+                seq = None # If no sequence is found for a component
+                if obj.sequences and len(obj.sequences) == 1:
+                    obj_seq = doc.find(obj.sequences[0])
+                    seq = Seq(obj_seq.elements.upper())
+                elif len(obj.sequences) > 1:
+                    raise ValueError(f"Component `{obj.display_id}` of given SBOL3 document has more than 1 sequnces (`{len(obj.sequences)}`). This is invalid; a component may only have 1 or 0 sequences.")
+                # TODO: "Version" annotation information currently not stored when converted genbank to sbol3
+                seq_rec = SeqRecord(
+                    seq=seq,
+                    id=obj.display_id,
+                    description=obj.description,
+                    name=obj.display_id,
+                )
+                # TODO: hardcoded molecule_type as DNA, derivation?
+                seq_rec.annotations["molecule_type"] = "DNA"
+                # TODO: hardcoded topology as linear, derivation?
+                seq_rec.annotations["topology"] = "linear"
+                # TODO: temporalily hardcoding version as "1"
+                # FIXME: Version still not being displayed on record's VERSION
+                seq_rec.annotations["sequence_version"] = self.DEFAULT_GB_REC_VERSION
+                seq_rec_features = []
+                if obj.features:
+                    # converting all sequence features
+                    for obj_feat in obj.features:
+                        # TODO: Also add ability to parse subcomponent feature type
+                        # Note: Currently we only parse sequence features from sbol3 to genbank
+                        if isinstance(obj_feat, sbol3.SequenceFeature):
+                            logging.info(
+                                f"Parsing feature `{obj_feat.name}` for component `{obj.display_id}`"
+                            )
+                            # TODO: There may be multiple locations for a feature from sbol3; 
```

Comment:
@jakebeal, the TODO is present here for multiple feature locations

Reply:
Huh... for some reason I was seeing only the old version. Thank you!